lib/src/mock_query_document_snapshot.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:cloud_firestore_mocks/src/mock_document_reference.dart';
+
+import 'mock_document_snapshot.dart';
+
+class MockQueryDocumentSnapshot extends MockDocumentSnapshot
```

Comment:
See: https://github.com/FirebaseExtended/flutterfire/blob/master/packages/cloud_firestore/cloud_firestore/lib/src/query_snapshot.dart

After 0.14, `QueryDocumentSnapshot ` is used.
for example, `QuerySnapshot #documents` returns `List<QueryDocumentSnaposhot>`.
See: https://github.com/FirebaseExtended/flutterfire/blob/7eb9e21f82a09c5cecebfc231b256545cf1ce073/packages/cloud_firestore/cloud_firestore/lib/src/query_snapshot.dart#L25